xwiki-platform-core/xwiki-platform-node/src/main/node/ds/api/etc/platform-dsapi.api.md

```diff
@@ -37,14 +34,12 @@ export type AbstractElements = {
     XMenu: DefineComponent<MenuProps & HTMLAttributes>;
     XMenuItem: DefineComponent<MenuItemProps & HTMLAttributes>;
     XMenuLabel: DefineComponent<MenuLabelProps & HTMLAttributes>;
-    XNavigationTree: DefineComponent<NavigationTreeProps & HTMLAttributes>;
-    XNavigationTreeSelect: DefineComponent<NavigationTreeSelectProps & HTMLAttributes>;
     XSelect: DefineComponent<SelectProps & HTMLAttributes>;
     XTab: DefineComponent<TabProps & HTMLAttributes>;
     XTabGroup: DefineComponent<HTMLAttributes & TabGroupProps>;
     XTabPanel: DefineComponent<TabPanelProps & HTMLAttributes>;
     XTextField: DefineComponent<TextFieldProps & ImgHTMLAttributes>;
-    XTree: DefineComponent<TreeProps & HTMLAttributes>;
+    XTree: DefineComponent<TreeProps<DisplayableTreeNode & any> & HTMLAttributes>;
```

Comment:
Isn't `TreeProps<DisplayableTreeNode & any>` equivalent to `TreeProps<any>`?

Reply:
Right, maybe add a comment in the source to explain the logic as I might not be the only one to be confused by this.